tests/integration/test_issue_1336.py

```diff
@@ -16,4 +17,24 @@ def test_pip_leak(tmpdir):
     python = ensure_python_interpreter(PY310)
     pip = os.path.join(os.path.dirname(python), "pip")
     subprocess.check_call(args=[pip, "install", "setuptools_scm==6.0.1"])
-    run_pex_command(args=["--python", python, "bitstring==3.1.7"], python=python).assert_success()
+
```

Comment:
This was hanging and made me suspect the new jobs join was hanging. Clearly this test was not running before though since the LHS is designed to hang! That Pex invocation neither output a PEX (-o) nor passed CLI args to the ephemeral PEX (--).